tests/scripts/all.sh

```diff
@@ -505,6 +505,30 @@ tests/ssl-opt.sh -f RSA
 msg "test: RSA_NO_CRT - RSA-related part of compat.sh (ASan build)" # ~ 3 min
 tests/compat.sh -t RSA
 
+# Re-run with reduced output message buffer
+msg "build: with ASan, small output buffer"
+cleanup
+cp "$CONFIG_H" "$CONFIG_BAK"
+scripts/config.pl set MBEDTLS_SSL_IN_CONTENT_LEN 16384
+scripts/config.pl set MBEDTLS_SSL_OUT_CONTENT_LEN 4096
+CC=gcc cmake -D CMAKE_BUILD_TYPE:String=Asan .
+make
+
+msg "test: ssl-opt.sh (ASan, MFL-related and large packet tests)"
+if_build_succeeded tests/ssl-opt.sh -f "Max fragment\|Large packet"
+
+# Re-run MFL tests with both buffers reduced
+msg "build: ASan, small total buffer"
```

Comment:
Same remark as above regarding leading with the important part. Also this isn't a *total* buffer, it's *both* buffers that are reduced.